.changeset/loud-cameras-hug.md

```diff
@@ -0,0 +1,6 @@
+---
+'@guardian/source-foundations': major
+'@guardian/source-react-components': major
```

Comment:
`@guardian/source-react-components-development-kitchen` is not being bumped, even though both these packages are `peerDependencies` and they are now both out of range. It's like changesets can't see the dev kitchen or is ignoring it for some reason 🤔

Reply:
From the changesets box above, it looks like all packages are going to be bumped... ?
![image](https://user-images.githubusercontent.com/15648334/169263217-9cc40464-137e-4e51-afd1-391ca5c7f684.png)